modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -57,6 +64,10 @@ class ApiClient(object):
     PRIMITIVE_TYPES = (
         (float, bool, six.binary_type, six.text_type) + six.integer_types
     )
+    ECDSA_MODE_FIPS_186_3 = 'fips-186-3'
+    ECDSA_MODE_DETERMINISTIC_RFC6979 = 'deterministic-rfc6979'
+    ECDSA_KEY_SIGNING_ALGORITHMS = {ECDSA_MODE_FIPS_186_3, ECDSA_MODE_DETERMINISTIC_RFC6979}
+
```

Comment:
Just a general question, not necessary to do:
What do you think about moving these constants and
- get_http_signature_headers
- get_message_digest
- sign_digest
- get_authorization_header
into a package called signing.py?
That way our users could use something like:
```
from petstore_api import signing
# Configure API client with HTTP signature authentication:
conf = petstore_api.Configuration(
  key_id='my-key-id',
  private_key_path='rsa.pem',
  signing_scheme=signing.hs2019,
  signing_algorithm=signing.PSS,
  signed_headers=['(request-target)', '(created)', 'host', 'date', 'Content-Type', 'Digest']
)
```
These constants and functions seem much more related to signing than the api_client
What do you think?

Reply:
Sure, I can do that. Where would you commit the signing.py file in GIT?

Reply:
Yup it would be a signing.mustache file that we would add in the templates folder and the .java code generator

Reply:
It may make sense to only conditionally use the template if we are using hasHttpSignatureMethods